web/test/pages/profile.test.tsx

```diff
@@ -2351,6 +2412,7 @@ describe("profile", () => {
           operatingPhase: "UP_AND_RUNNING_OWNING",
           dateOfFormation: undefined,
           existingEmployees: undefined,
+          legalStructureId: randomPublicFilingLegalStructure(),
```

Comment:
Had to change it to have only randomPublicFilingLegalStructure() because some of the legal structures would make it fail.